ml-proto/spec/kernel.ml

```diff
@@ -119,7 +119,7 @@ and func' =
 type memory = memory' Source.phrase
 and memory' =
 {
-  initial : Memory.size;
+  min : Memory.size;
```

Comment:
It's perhaps subtle, but why is "initial" changing to "min" here and elsewhere? Apparently BinaryFormat.md calls it "minimum", but AstSemantics.md still calls it "initial".